sdk/communication/azure-communication-sms/README.md

```diff
@@ -128,15 +132,34 @@ for (SmsSendResult result : sendResults) {
 
 ## Troubleshooting
 
-All SMS service operations will throw an exception on failure.
-<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L104-L112 -->
+- SMS operations will throw an exception if the request to the server fails.
+- Exceptions will not be thrown if the error is caused by an individual message, only if something fails with the overall request.
+- Please use the `Successful` flag to validate each individual result to verify if the message was sent.
+<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L248-L273 -->
 ```java
+SmsClient smsClient = createSmsClientUsingAzureKeyCredential();
+
+SmsSendOptions options = new SmsSendOptions();
+options.setDeliveryReportEnabled(true);
+options.setTag("Tag");
+
 try {
-    SmsSendResult sendResult = smsClient.send(
+    Response<Iterable<SmsSendResult>> sendResults = smsClient.sendWithResponse(
         "<from-phone-number>",
-        "<to-phone-number>",
-        "Hi"
-    );
+        Arrays.asList("<to-phone-number1>", "<to-phone-number2>"),
+        "Hi",
+        options /* Optional */,
+        Context.NONE);
+
+    Iterable<SmsSendResult> resultOfEachMessage = sendResults.getValue();
+    for (SmsSendResult result : resultOfEachMessage) {
+        if (!result.isSuccessful()) {
+            System.out.println("Successfully sent this message: " + result.getMessageId() + " to " + result.getTo());
+        } else {
+            System.out.println("Something went wrong when trying to send this message " + result.getMessageId() + " to " + result.getTo());
```

Comment:
Let's print out the status code and the error message as well. Refer to C# README